File: app/src/main/java/com/example/vamshi/homwayprojectchallenge/Screens/MainScreen/ContractPresenterView.java
package com.example.vamshi.homwayprojectchallenge.Screens.MainScreen;

public interface ContractPresenterView {

    interface PresenterMainWork {

        void getPlaces(String s);

        void addFavourite(String key);

        void removeFavourite(String key);

        Boolean isFavourite(String key);

        void maStart();

        void maStop();

    }

}

File: app/src/main/java/com/example/vamshi/Curious/Screens/MainScreen/ContractPresenterView.kt
package com.example.vamshi.Curious.Screens.MainScreen

interface ContractPresenterView {

    interface PresenterMainWork {

        fun getPlaces(s: String): Boolean

        fun addFavourite(key: String)

        fun removeFavourite(key: String)

        fun isFavourite(key: String): Boolean

        fun maStart()

        fun maStop()
    }
}
